wiki/FitNesseRoot/HsacExamples/TaggedTestcases.wiki

```diff
@@ -0,0 +1,23 @@
+---
+Suite
+---
+!1 Suite To test running a filtered suite from maven
+
+Running this suite using: 
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DsuiteFilter=runMe -DexcludeSuiteFilter=dontRun
+  * Results in 3 test cases 001, 002 to be executed (all cases that have the tag 'runMe', and don't have the tag 'dontRun')
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DexcludeSuiteFilter=runMe
+  * Results in 3 test cases 003, 004, 006 to be executed (all cases that don't have the tag 'runMe') 
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DsuiteFilter=runMe
+  * Results in 3 test cases 001, 002, 005 to be executed (all cases that do have the tag 'runMe')
+  
+Passing a comma separated list to the (exlude)suiteFilter is allowed, so we can also use:
```

Comment:
I can make the suiteFilter configurable (so we can use a suiteFilterStrategy sysprop to set it to AND). This works, because FitNesse supports AND/OR for suiteFilters. For excludeSuiteFilter however, Fitnesse only supports OR. See SuiteFilter constructor:

`public SuiteFilter(String orTags, String mustNotMatchTags, String andTags, String startWithTest) {
    this.startWithTest = (!"".equals(startWithTest)) ? startWithTest : null;
    if(andTags != null) {
      matchTags = new SuiteTagMatcher(andTags, true, true);
    } else {
      matchTags = new SuiteTagMatcher(orTags, true, false);
    }
    notMatchTags = new SuiteTagMatcher(mustNotMatchTags, false, false);
  }`

The only way I see to get the excludeFilter to be configurable would be overriding most of the SuiteFilter class with a custom (HsacSuiteFilter) class, which would cause a lot of private method duplication. I'd opt to keep this filter's behaviour similar to the behaviour in Fitnesse's suite responder. (configurable AND/OR for tags to include, OR for tags to exclude). Agree?